Area.cpp

```diff
@@ -0,0 +1,37 @@
+#ifndef Area_cpp
+#define Area_cpp
+
+#include <iostream>
+
+using namespace std;
+
+void CaculateSquareArea(){
+    int length, width, area;
+    cout <<"Enter square dimensions" <<endl;
+    cout <<"Enter Length" <<endl;
```

Comment:
This method name suggests it will only calculate area of the square, but it is also accepting input from the user.